package.json

```
@@ -16,7 +16,7 @@
     "post-compile": "rimraf dist/*.test.* dist/**/*.test.* dist/**/*.stories.* dist/docs dist/json dist/assets"
   },
   "dependencies": {
-    "@ukhomeoffice/cop-react-components": "^0.8.1",
+    "@ukhomeoffice/cop-react-components": "^0.8.2",
```

The current version of cop-react-components is 1.1.2, should this be reflected here?
```suggestion
    "@ukhomeoffice/cop-react-components": "^1.1.2",
```

---

Not in this one or the unit tests will fail. It's in this PR:
https://github.com/UKHomeOffice/cop-react-form-renderer/pull/59/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R19

---

Ah yes, just had a look at the other PR and seen its updated.